schema.sql

```diff
@@ -1,5 +1,43 @@
 /* Database schema to keep the structure of entire database. */
 
+PSQL
+CREATE DATABASE vet_clinic /* Creating vet clinic database */
+\c vet_clinc /* Connect to the vet_clinic db using psql*/
 CREATE TABLE animals (
-    name varchar(100)
-);
\ No newline at end of file
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth DATE,
+    escape_attempts INTEGER,
+    neutered BOOLEAN,
+    weight_kg DECIMAL(10, 2)
+);
+/* Create column called species */
+ALTER TABLE animals
+ADD species VARCHAR(255);
+
+
+psql
+/c mydatabases
+CREATE TABLE owner (
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(255),
+  age INTEGER
+);
+
```

Comment:
- [ ] Kindly ensure you're  consistent on your naming. You wrote `owner` here and you wrote `owners` while trying to insert data into this table and that affects the populate of that table.

![owners](https://github.com/LDouglasOT/vet-clinic-dbs/assets/26597432/0bd7fd54-adc3-47e9-a299-d7e7c2008bf8)